Backend/ServerModules/ServerBehavior.cs

```diff
@@ -68,5 +68,24 @@ protected override void OnOpen()
             UserManager.Connect(socket, ip);
             CLogger.Event($"New client connected from: {ip}");
         }
+
+        protected override void OnClose(CloseEventArgs e)
+        {
+            WebSocket socket = Context.WebSocket;
+            User? user = UserManager.GetUserBySocket(socket);
+            if (user == null)
+            {
+                CLogger.Error("Disconnected user not found in UserList");
+                return;
+            }
+
+            UserManager.UsersList.Remove(user);
+            CLogger.Event($"User disconnected: {user.Username}");
+
+            TextMessage disconnectAlert = new TextMessage(null, "");
+            disconnectAlert.Content = $"User {user.Username} has disconnected";
```

Comment:
Make a function called SendAlert that accepts  string in LSMPBehavior

Reply:
Done